api/v1alpha1/etcdpeer_types.go

```diff
@@ -61,7 +61,7 @@ type EtcdPeerStorage struct {
 	// StorageClass and the Storage Resource Request in this template.
 	// That PVC will then be mounted in the Pod for this EtcdPeer and the Etcd
 	// process when it starts will persist its data to the PV bound to that PVC.
-	VolumeClaimTemplate *corev1.PersistentVolumeClaimSpec `json:"volumeClaimTemplate,omitempty"`
+	VolumeClaimTemplate corev1.PersistentVolumeClaimSpec `json:"volumeClaimTemplate,omitempty"`
```

Comment:
I think this is a pointer because we may also add `emptyDir` as an option in future? I'm not sure. Basically, this should only be a non pointer field if it is required: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required

Reply:
It is required, right now.